{{cookiecutter.repo_name}}/package.json

```diff
@@ -3,107 +3,86 @@
-    "http-proxy-middleware": "^0.18.0",
-    "husky": "^3.0.1",
-    "imports-loader": "^0.8.0",
-    "jsdom": "^11.6.2",
-    "koa-connect": "^2.0.1",
-    "lint-staged": "^9.2.0",
-    "lynt": "^0.5.5",
-    "prettier": "^1.18.2",
-    "webpack-bundle-analyzer": "^3.6.0",
-    "webpack-dev-server": "^3.1.10"
+    "@testing-library/jest-dom": "^5.5.0",
+    "@testing-library/react": "^10.0.3",
+    "babel-eslint": "^10.1.0",
+    "babel-jest": "^26.0.1",
+    "cypress": "^4.8.0",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
+    "enzyme-to-json": "^3.4.4",
+    "eslint": "^7.0.0",
+    "eslint-config-airbnb": "^18.1.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.3",
+    "eslint-plugin-react": "^7.20.0",
+    "husky": "^4.2.5",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^26.0.1",
+    "jest-watch-typeahead": "^0.6.0",
+    "lint-staged": "^10.2.4",
+    "miragejs": "^0.1.38",
+    "prettier": "^2.0.5",
+    "react-test-renderer": "^16.13.1"
   },
   "dependencies": {
-    "@swup/ga-plugin": "^1.0.1",
-    "@swup/preload-plugin": "^1.0.3",
-    "autoprefixer": "^8.2.0",
-    "babel": "^6.5.2",
-    "babel-core": "^6.13.2",
-    "babel-loader": "^7.1.4",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-plugin-transform-es2015-destructuring": "^6.23.0",
-    "babel-plugin-transform-es2015-parameters": "^6.24.1",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "browser-sync-webpack-plugin": "^2.2.2",
-    "classnames": "^2.2.5",
-    "connect-history-api-fallback": "^1.5.0",
-    "css-loader": "^0.28.11",
-    "file-loader": "^1.1.11",
-    "foundation-sites": "^6.4.4-rc1",
-    "fullcalendar": "^3.9.0",
-    "fullcalendar-scheduler": "^1.9.4",
-    "glob": "^7.1.2",
-    "jest": "^23.6.0",
-    "jquery": "^3.3.1",
-    "jquery-datepicker": "^1.12.3",
-    "jquery-ui": "^1.12.1",
-    "mini-css-extract-plugin": "^0.4.0",
-    "moment": "^2.22.1",
-    "node-sass": "^4.11.0",
-    "normalize.css": "^8.0.0",
-    "nunjucks": "^3.1.2",
-    "nunjucks-loader": "^3.0.0",
-    "postcss-clean": "^1.1.0",
-    "postcss-loader": "^2.1.3",
-    "resolve-url-loader": "^2.3.0",
-    "sass-loader": "^6.0.7",
-    "select2": "^4.0.6-rc.1",
-    "style-loader": "^0.20.3",
-    "swup": "^2.0.5",
-    "uglifyjs-webpack-plugin": "^1.2.6",
-    "url-loader": "^1.0.1",
-    "webpack": "^4.4.1",
-    "webpack-bundle-tracker": "0.3.0",
-    "webpack-cli": "^3.1.1",
-    "webpack-md5-hash": "0.0.6",
-    "webpack-merge": "^4.1.2"
+    "@date-io/date-fns": "^2.5.0",
+    "@material-ui/core": "^4.9.10",
+    "@material-ui/icons": "^4.9.1",
+    "@sentry/browser": "^5.15.5",
+    "@sentry/node": "^5.15.5",
+    "@sentry/webpack-plugin": "^1.11.1",
+    "axios": "^0.19.2",
+    "clsx": "^1.1.0",
```

Comment:
We should add a section to readme with libraries people should use for X reasons. This is one that I would not remember to use, personally, so adding to readme might help? I thought we already had a spot for it.